sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -1694,6 +1747,40 @@ public partial class CassandraTableResourceInfo
         public Azure.ResourceManager.CosmosDB.Models.CassandraSchema Schema { get { throw null; } set { } }
         public string TableName { get { throw null; } set { } }
     }
+    public partial class ClientEncryptionIncludedPath
+    {
+        public ClientEncryptionIncludedPath(string path, string clientEncryptionKeyId, string encryptionType, string encryptionAlgorithm) { }
+        public string ClientEncryptionKeyId { get { throw null; } set { } }
+        public string EncryptionAlgorithm { get { throw null; } set { } }
+        public string EncryptionType { get { throw null; } set { } }
+        public string Path { get { throw null; } set { } }
+    }
+    public partial class ClientEncryptionKeyGetPropertiesResource : Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource
+    {
+        public ClientEncryptionKeyGetPropertiesResource() { }
+        public Azure.ETag? ETag { get { throw null; } }
+        public string Rid { get { throw null; } }
+        public float? Timestamp { get { throw null; } }
+    }
+    public partial class ClientEncryptionKeyGetResultCreateOrUpdateContent
+    {
+        public ClientEncryptionKeyGetResultCreateOrUpdateContent(Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource resource) { }
+        public Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyResource Resource { get { throw null; } }
+    }
+    public partial class ClientEncryptionKeyResource
+    {
+        public ClientEncryptionKeyResource() { }
+        public string EncryptionAlgorithm { get { throw null; } set { } }
+        public string Id { get { throw null; } set { } }
+        public Azure.ResourceManager.CosmosDB.Models.KeyWrapMetadata KeyWrapMetadata { get { throw null; } set { } }
+        public byte[] WrappedDataEncryptionKey { get { throw null; } set { } }
+    }
+    public partial class ClientEncryptionPolicy
```

Comment:
Shall we rename this to `CosmosDBClientEncryptionPolicy`?

Reply:
If we are uniform all across for example IndexingPolicy etc. I am ok with it. Is IndexingPolicy being renamed to CosmosDBIndexingPolicy? Same w.r.t other comments regarding  renaming.

Reply:
@kr-santosh please take a look at this.

Reply:
is now CosmosDBClientEncryptionPolicy

Reply:
Does using rename-model: suffice. Do have a look at the changes made.

Reply:
@ArcturusZhang  do have a look at the changes

Reply:
#resolved